package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "floeproject.org",
+  "version": "1.0.0",
+  "description": "FLOE Project Website",
+  "main": "index.js",
+  "scripts": {
+    "build": "eleventy",
+    "eleventy": "eleventy --serve",
+    "eleventyport": "eleventy --serve --port=9778"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fluid-project/floeproject.org.git"
+  },
+  "keywords": [
+    "Floe",
+    "Project",
+    "Flexible",
+    "Learning",
+    "for",
+    "Open",
+    "Education"
+  ],
+  "author": "OCAD University <idrc@ocadu.ca>",
+  "contributors": [
+    "Sachin Chopra <schopra@mt.iitr.ac.in> (www.github.com/sachin10101998)"
+  ],
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fluid-project/floeproject.org/issues"
+  },
+  "homepage": "https://github.com/fluid-project/floeproject.org#readme",
+  "dependencies": {
+    "@11ty/eleventy": "^0.11.0",
+    "@11ty/eleventy-plugin-rss": "^1.0.7",
+    "@11ty/eleventy-plugin-syntaxhighlight": "^3.0.1",
+    "@tbranyen/jsdom": "^13.0.0",
+    "concurrently": "^5.2.0",
+    "grunt": "^1.1.0",
+    "grunt-contrib-clean": "^2.0.0",
+    "grunt-contrib-copy": "^1.0.0",
+    "html-minifier": "^4.0.0",
+    "image-size": "^0.8.3",
+    "infusion": "^3.0.0-dev.20200326T173810Z.24ddb2718",
+    "sanitize.css": "^11.0.1",
+    "slugify": "^1.4.0"
+  },
```

Comment:
Please review if these are actually dev dependencies. That is are they only needed for testing, building and etc. Even for things that are included in the final output, but are copied over, like Infusion, could probably be moved to a dev dependency.

Reply:
Fixed